### ISSUE TITLE
Watch should support rest-src

### ISSUE THREAD

**Issue by eurochriskelly** [fgeorges/mlproj-core#25]
Given a named source with type=rest-src, it should watch the same way it deploys:
```
sources: [{
    name: 'foo',
    type: 'rest-src',
    dir: 'foo/'
}]
```
The command `npm deploy -s foo` registers the rest services correctly.
The command `npm watch -s foo` just updates the services.

**Comment by fgeorges** [comment]
@eurochriskelly Should be fixed in 0.46.6 + 0.28.1, can you validate and close the ticket, please?

Note filtering from the source set is not applied yet in case of watch (so e.g. editor backup files will display an error, but the watch command itself will work).  This is the topic of #26.

**Comment by eurochriskelly** [comment]
This appears to work nicely now!
